src/lucky/tags/forgery_protection_helpers.cr

```diff
@@ -5,21 +5,21 @@ module Lucky::ForgeryProtectionHelpers
   # `Lucky::FormHelpers#form_for`. It creates a hidden input with the CSRF
   # token. THis ensures that the form is safe. If you try to submit a form
   # without a CSRF token it will fail with a 403 forbidden status code.
-  def csrf_hidden_input
+  def csrf_hidden_input(context : HTTP::Server::Context)
```

Comment:
It looks like this will need to be changed in https://github.com/luckyframework/lucky/blob/500e4bc28786722700672c398b768b3937b204cd/src/lucky/tags/form_helpers.cr#L9

which also makes me wonder if we should pass in the context for this. That would mean that we'd need to pass in the context to `form_for` so that it could then get passed to `csrf_hidden_input`. That seems like a bit of a pain. Any ideas? Maybe we leave this one as it was?

Reply:
hmm.. Well my first concern is why didn't this get caught when I ran the specs? Shouldn't the compiler have thrown a fit? Secondly, that does make things a bit of a pain. My first thought was just pass in `@context`, but then it becomes magical again just in a different spot. We could leave it off, but that's also a little confusing. 3 months down the road someone will look at the source code and assume that's a mistake. I'll forget, and then merge that mistake lol.

Reply:
Maybe I inadvertently created an issue that wasn't really an issue because I was reading too much in to the code? 🤔 We could just close this PR, and wait for someone else to bring it up?

Reply:
I'm not sure why it didn't catch it locally. Travis caught it so that is super weird. Maybe some caching issue?

And yeah this is a tricky one...I'm almost thinking we keep things as-is so that the API is similar.

Another thought though...we could see what it would take to always pass the `context` to all components. I'm not sure how that'd work or if that'd be a pain to test but that might be a way around it too.

Reply:
Yeah let's keep it as-is for now. But if we can think of a good way to clean this up w.o introducing other issues I'm down!